pkg/cmd/kafka/create/create.go

```diff
@@ -518,29 +519,29 @@ func checkForLegacyQuota(opts *options, orgQuotas *accountmgmtutil.OrgQuotas) {
 	}
 
 	// Check if the user has a legacy quota
-	for _, quota := range orgQuotas.TrialQuotas {
+	for _, quota := range orgQuotas.StandardQuotas {
 		if quota.Quota > 0 {
 			opts.hasLegacyQuota = true
 			return
 		}
 	}
-
-	// Check if the user has a legacy quota
-	for _, quota := range orgQuotas.StandardQuotas {
+	for _, quota := range orgQuotas.MarketplaceQuotas {
 		if quota.Quota > 0 {
 			opts.hasLegacyQuota = true
 			return
 		}
 	}
-	for _, quota := range orgQuotas.MarketplaceQuotas {
+	for _, quota := range orgQuotas.EvalQuotas {
 		if quota.Quota > 0 {
 			opts.hasLegacyQuota = true
 			return
 		}
 	}
-	for _, quota := range orgQuotas.EvalQuotas {
+
+	// Check if the user has a trial quota, if so they ONLY have trial quota
+	for _, quota := range orgQuotas.TrialQuotas {
 		if quota.Quota > 0 {
-			opts.hasLegacyQuota = true
+			opts.useTrialFlow = true
```

Comment:
Didn't we find out trial is always 0 and it is only known if they have trial quota left if they have no trial instances running?

Reply:
yes you are correct. I should have checked the changes, more thoroughly. give me a min and I'll correct 👍